package.json

```diff
@@ -16,15 +17,15 @@
     "prop-types": "^15.7.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-scripts": "^4.0.3",
+    "vite": "^2.3.2",
+    "vite-plugin-eslint": "^1.3.0",
```

Comment:
I believe Vite is only required as a dev dependency. See https://github.com/vitejs/create-vite-app/blob/master/template-react/package.json

Reply:
But, because use prettier "..." so it should be listed on dependencies :) is it right? because i got error like that
![Screenshot from 2021-08-23 15-42-49](https://user-images.githubusercontent.com/57583780/130417750-1deb79f1-68d8-4005-93db-81ac886b3951.png)

Reply:
I think it's caused by prettier thinks that `vite.config.js` is a production source files, but it's actually a config file, just like `.eslintrc.js` which wouldn't (and shouldn't) be included in the production. Either you could ignore that file, or only ignore lines that caused that linter error (`import { defineConfig } from "vite";`). See https://prettier.io/docs/en/ignore.html.